packages/legend-studio/src/stores/sidebar-state/testable/GlobalTestRunnerState.ts

```diff
@@ -496,14 +496,11 @@ export class GlobalTestRunnerState {
       .filter(isNonNullable);
   }
 
-  init(force?: boolean): void {
-    if (!this.testableStates || force) {
-      const testables =
-        this.editorStore.graphManagerState.graph.allOwnTestables;
-      this.testableStates = testables.map(
-        (testable) => new TestableState(this.editorStore, this, testable),
-      );
-    }
+  init(): void {
```

Comment:
This `int` is called within a `useEffect`, this would cause the whole test runner to refresh upon each render. 
Might make sense to alter the useEffect logic rather than change this logic.